db/utils.js

```diff
@@ -126,6 +138,10 @@ const DBUtils = {
   getBreachesForEmail(email) {
     return this.getBreachesForHash(getSha1(email));
   },
+
+  async getBreachByName(breachName) {
+    return (await Breach.query().where("name", breachName))[0];
```

Comment:
Looks like Objections `QuerySet` objects handle this for us with some kind of fancy getter that returns `undefined` instead of throwing an error?
```
> breaches
[  ]
> breaches[0]
undefined
```

Reply:
Ah right, we are accessing an element that may not exist but we're not doing anything with it so it doesn't need a null check here - that's up to the consumer of this function. 👍